docs/design-proposals/2020-09-27-tidb-monitor-refactor.md

```diff
@@ -0,0 +1,49 @@
+# TiDB Monitor Refactor
+
+## Summary
+
+This document presents a design to refactor tidb monitor,solve some problems in monitoring uniformly.
+
+## Motivation
+
+Currently, it is not friendly that tidbmonitor monitor multiple cluster under the same namespaces or different namespaces.In a multi-cluster TLS environment, the cert of tidbmonitor is bound to the first cluster of the cluster,we should eliminate this binding relationship.
+Secondly,tidbmonitor is also a statefulset application,we should switch to statefulset deploy from deployment crd,this process should smooth upgrade.
+Also, many users will integrate the company’s existing monitoring system, we should support thanos sidecar spec refer to the Prometheus operator, and optimize metric service and provide googe yaml example like `ServiceMonitor` ,`thanos` etc.
+Others are some small optimization points,providing more field for `kubectl get tidbmonitor`  command, supporting multiple cluster grafana dashborad,optimizing pd dashboard address writing logic,fulling e2e and unit testing etc.
+
+### Goals
+
+* Make monitoring easier to use.
+
```

Comment:
I think the targets could be more specific, eg: Make TidbMonitor able to monitor multiple TidbClusters.

Reply:
ok